win32/resource.rb

```diff
@@ -61,6 +61,7 @@ def icons.find(path)
 #include <winver.h>
 
 #{icon || ''}
+1 24 "win32/ruby.manifest"
```

Comment:
Seems DLL doesn't need the manifest.

```suggestion
#{type == 'VFT_APP' ? "1 RT_MANIFEST ruby.manifest" : ""}
```